project_app/components/Footer.js

```diff
@@ -7,17 +7,38 @@ import {
   ProductContainer,
 } from "./styled";
 import Image from "next/image";
+import Link from "next/link";
 
 export default function Footer() {
-  const titles = ["About Us", "FAQ", "Contact Us", "Give Feedback"];
-
+  const titles = [
+    {
+      name: "About",
+      link: "/About",
+    },
+    {
+      name: "FAQ",
+      link: "/Faq",
+    },
```

Comment:
hi no, this is not what i meant. 
they should be lowercase
/about, /faq
then your filenames should be
about.tsx, faq.tsx

Reply:
you wouldnt see urls with capitalized initials

Reply:
This has no caps in it -